R/apa_print_lmerMod.R

```diff
@@ -0,0 +1,27 @@
+apa.print.lmerMod <- function(
+  lmerMod_fit
+  , conf.level = 0.95
+  , ...
+){
+  
+  tidy_lmerMod_fit <- broom.mixed::tidy(lmerMod_fit, conf.int=TRUE, conf.level=conf.level)
+  
+  tidy_fixed_eff <- subset(as.data.frame(tidy_lmerMod_fit), effect=="fixed")
+  tidy_random_eff <- subset(as.data.frame(tidy_lmerMod_fit), effect=="ran_pars")
+  
+  tidy_fixed_eff <- tidy_fixed_eff %>% rename("Effect" = "term"
```

Comment:
This is a great start. Maybe a nice next step would be to add variable labels using [`variable_labels()`](https://github.com/crsh/papaja/blob/devel/vignettes/add_apa_print_methods.Rmd#L100).